packages/cozy-harvest-lib/src/helpers/accounts.js

```diff
@@ -94,6 +96,55 @@ export const mergeAuth = (account, authData) => ({
   auth: merge(account.auth, authData)
 })
 
+/**
+ * Gets the vault cipher relationship to an account
+ * @param  {object} account   io.cozy.accounts document
+ * @return {string}           cipher uuid
+ */
+export const getVaultCipherId = account =>
+  account.relationships &&
+  account.relationships.vaultCipher &&
+  account.relationships.vaultCipher._id
+
+/**
+ * Gets the linked cipher in vault
+ * If none is linked, search on that match
```

Comment:
This function does 2 pretty different things, do you think it would make sense to split it into 2 separate functions?